l10n_it_abicab/abicab.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2012
+#    Associazione OpenERP Italia (<http://www.openerp-italia.org>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#############################################################################
+
+from openerp import models, fields, api
+
+
+class ResBank(models.Model):
+
+    _inherit = "res.bank"
+
+    abi = fields.Char(size=5, string='ABI')
+    cab = fields.Char(size=5, string='CAB')
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = "res.partner.bank"
+
+    bank_abi = fields.Char(size=5, string='ABI')
+    bank_cab = fields.Char(size=5, string='CAB')
+
+    @api.onchange('bank')
+    @api.cr_uid_ids_context
```

Comment:
The original definition of onchange in module 'base' of Odoo is not porting to 8 yet.
Like this work in any version